code/blake2/Hacl.Blake2b_256.fst

```diff
@@ -51,3 +51,6 @@ let load_state256b_from_state32: Core.load_state_st Spec.Blake2B Core.M256 =
   Core.load_state_from_state32 #Spec.Blake2B #Core.M256
 let store_state256b_to_state32: Core.store_state_st Spec.Blake2B Core.M256 =
   Core.store_state_to_state32 #Spec.Blake2B #Core.M256
+
```

Comment:
this needs to be marked private, so that it appears in the internal header instead of the public one

Reply:
let's do it as part of a followup PR, when we fix bundling for Blake2 in #693 -- this should be feasible easily with something like `-bundle Hacl.Streaming.Blake2b=Hacl.Blake2b` -- right now making it private obviously prevents EverCrypt.Hash from referring to this